libbpf-rs/src/map.rs

```diff
@@ -409,16 +426,81 @@ impl Map {
     pub fn keys(&self) -> MapKeyIter {
         MapKeyIter::new(self, self.key_size())
     }
+
+    /// Create the bpf map standalone.
+    pub fn create_map(
+        name: &str,
```

Comment:
I think defining the type here as AsRef<str> allows for more generic usages. In other libbpf-rs functions, we pass in AsRef<str>s.

Reply:
sounds reasonable to me.